Sources/Path.swift

```diff
@@ -119,38 +98,42 @@ public struct Path {
         }
     }
 
-    /// The components of the path.
-    ///
-    /// Return [] if path is `.` or `""`
+    /**
+     The components of the path.
+
+     Returns [] if path is `.` or `""`
+    */
     public var components: [Path] {
-        if rawValue == "" || rawValue == "." {
+        guard rawValue != "" && rawValue != "." else {
```

Comment:
`guard` is not always the solution

I see a lot `if` replaced by `guard` with tricky boolean conditions

it's a matter of coding style, `if` has been clear here, `guard` also is clear